### PR TITLE
Make mysql techspec compliant 

### PR DESCRIPTION
**Pull request by epwn**
Delete test database after installation
 
Setup logrotation

Fix the location of general and error log files for mysql

Fix a typo in monitoring.yml

### CODE REVIEW

**Comment by twaldrop** [comment]
ok to test

File: roles/percona-server/tasks/main.yml [hunk 1]
```diff
@@ -66,6 +66,11 @@
 - name: install root .my.cnf defaults file
   template: src=root/.my.cnf dest=/root/.my.cnf owner=root group=root mode=0600
 
+- name: remove mysql test database
+  mysql_db: state=absent name=test
+  notify:
+    - restart mysql server
```
Comment (j2sol) [inline roles/percona-server/tasks/main.yml]: Why would we need to restart the daemon after deleting a database?

Comment (pwplusnick) [inline roles/percona-server/tasks/main.yml]: We don't. I'll hit up ryan on slack to make the change.

Comment (epwn) [inline roles/percona-server/tasks/main.yml]: good observation. I thought it was just in good practice but i will make those changes and re-commit

**Comment by epwn** [comment]
Please hold off on merging. Some changes will need to be made. Label as needswork

**Comment by epwn** [comment]
needswork. please label it

**Comment by twaldrop** [comment]
ok to test

**Pull request merged**